screwdriver/screwdriver.go

```diff
@@ -41,11 +42,39 @@ type Step struct {
 	Command string `json:"command"`
 }
 
+// PairSlice is an associative array which preserves order
+type PairSlice struct {
+	Body []struct {
+		Key   string
+		Value string
+	}
+}
+
+// UnmarshalJSON replaces JSON of a normal associative array to PairSlice
+func (MS *PairSlice) UnmarshalJSON(data []byte) error {
```

Comment:
Why the name of reciver is `MS`?
Is it better to use `p` or `ps`?

Reply:
I forgot to change the name. I fixed it.